packages/origin-ui-core/src/components/bundles/SelectedForSale.tsx

```diff
@@ -76,49 +75,62 @@ export const SelectedForSale = (props: IOwnProps) => {
+                                        <Grid item xs={8}>
+                                            <ListItemText
+                                                primary={
+                                                    <div>
+                                                        <Typography>
+                                                            {province}, {region}
+                                                        </Typography>
+                                                        <Typography>{facilityName}</Typography>
+                                                    </div>
+                                                }
+                                                secondary={moment
+                                                    .unix(creationTime)
+                                                    .format('MMM, YYYY')}
+                                            />
+                                        </Grid>
+                                        <Grid item xs={4}>
+                                            <ListItemText
+                                                primary={EnergyFormatter.format(
+                                                    energy.toNumber(),
+                                                    true
+                                                )}
+                                                secondary={`${(
+                                                    (100 * energy.toNumber()) /
+                                                    totalVolume.toNumber()
+                                                ).toFixed(0)} %`}
+                                            />
+                                        </Grid>
                                     </Grid>
-                                    <Grid item xs={4}>
-                                        <ListItemText
-                                            primary={PowerFormatter.format(energy.toNumber(), true)}
-                                            secondary={`${(
-                                                (100 * energy.toNumber()) /
-                                                totalVolume.toNumber()
-                                            ).toFixed(0)} %`}
-                                        />
-                                    </Grid>
-                                </Grid>
-                            </ListItem>
-                        );
-                    })}
+                                </ListItem>
+                            );
+                        }
+                    )}
                 </List>
                 <Grid container justify="space-between">
                     <Grid item>Total Volume</Grid>
-                    <Grid item>{PowerFormatter.format(totalVolume.toNumber(), true)}</Grid>
+                    <Grid item>{EnergyFormatter.format(totalVolume.toNumber(), true)}</Grid>
```

Comment:
Please don't cast `totalVolume` to number. This might cause issues with big numbers. Forward just `totalVolume` to EnergyFormatter.